src/js/components/dashboard/DashboardPage.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+  * AgencyDataDashboardPage.jsx
+  * Created by Lizzie Salita 9/24/19
+  */
+
+import React from 'react';
+import Navbar from 'components/SharedComponents/navigation/NavigationComponent';
+import Footer from 'components/SharedComponents/FooterComponent';
+import Banner from 'components/SharedComponents/Banner';
+import DashboardTab from './DashboardTab';
+import FilterSidebar from './FilterSidebar';
+
+const filters = [
+    {
+        name: 'Quarter',
+        required: true,
+        component: null,
+        description: 'Select the applicable quarter(s).'
+    },
+    {
+        name: 'Fiscal Year',
+        required: true,
+        component: null,
+        description: 'Select the applicable fiscal year(s).'
+    },
+    {
+        name: 'Files',
+        required: true,
+        component: null,
+        description: 'Select one file or cross-file.'
+    },
+    {
+        name: 'Rule',
+        required: false,
+        component: null,
+        description: 'Enter specific rules to narrow the data.'
```

Comment:
This doesn't match the invision copy

Reply:
Thanks, must have changed since I last looked at the mock!